common/src/main/java/org/corfudb/common/compression/ZSTDCompression.java

```diff
@@ -37,6 +42,12 @@ public ByteBuffer compress(ByteBuffer uncompressed) {
                 uncompressed.array(), uncompressed.position(), uncompressed.remaining(),
                 DEFAULT_COMPRESSION_LEVEL);
 
+        double compressionRatio = (double) decompressedLength/compressedLen;
+        if (compressionRatio > EXPECTED_COMPRESSION_RATIO) {
+            MicroMeterUtils.measure(compressionRatio, "logdata.compression.ratio");
```

Comment:
It's fine to only log warning messages when ratio is high. But for metrics I think it should cover all compression ratios. Same for the rest metrics.